### ISSUE TITLE
Please add an option to hide the new bits integration to rewards

### ISSUE THREAD

**Issue by TsukiZero** [FrankerFaceZ/FrankerFaceZ#1508]
Namely hide the Power Ups options and the bit counter as seen here (please pay no mind to how I used Stylebot to move the reward popup to the side of the chat):
![image](https://github.com/FrankerFaceZ/FrankerFaceZ/assets/78157224/fcd801ee-2950-4258-9740-d3959bece5de)

**Comment by pointydev** [comment]
Personally I'd like to see separate options to hide the bits display next to channel points counter below the chat bar and hiding/reordering the power-ups redeemables in the menu itself.

For the former, this _(terrible, and probably soon to break)_ CSS can be used to hide the bits counter (apply using e.g. Stylus):
```css
/* Hide bits in channel points mini-display */
.community-points-summary:has([d="m3 12 7-10 7 10-7 6-7-6zm2.678-.338L10 5.487l4.322 6.173-.85.728L10 11l-3.473 1.39-.849-.729z"]) [data-a-target] > div {
    &> div:first-of-type,
    &> [data-test-selector="balance-string"] {
        display: none!important;
    }
}
```

**Comment by SephirothWS** [comment]
Yes please. I would LOVE to remove any and all mentions of bits from the channel points menu and revert it back to the way it was prior to them adding these permanently-marked-as-new-give-us-money-to-use options.

**Comment by simoniz0r** [comment]
I was able to restore the channel points menu back to how it looked before the power ups were added with the following ublock origin filter:

```
www.twitch.tv##div.ZvPWT.Layout-sc-1xcs6mc-0:nth-of-type(3), div.reward-list-item.iqUbUe.Layout-sc-1xcs6mc-0:nth-of-type(4), div.reward-list-item.iqUbUe.Layout-sc-1xcs6mc-0:nth-of-type(5), div.reward-list-item.iqUbUe.Layout-sc-1xcs6mc-0:nth-of-type(6)
```

Not sure how robust this will be, but it *should* be feasible to add an option to do this through FFZ.
